package.json

```
@@ -13,13 +13,12 @@
     "dev": "nodemon --config nodemon.config.json -x 'pnpm build'",
     "build": "turbo run build --filter=!docs --filter=!template-* --filter=!demo-* --cache-dir=.turbo",
     "build:all": "turbo run build --cache-dir=.turbo",
-    "ci:test": "./scripts/tests-ci.sh",
     "pretest": "turbo run pretest",
     "depcheck": "knip --dependencies --tags=-knipignore",
     "knip:fix": "knip --fix",
     "depsync:lint": "syncpack list-mismatches",
     "depsync:fix": "syncpack fix-mismatches",
-    "test": "vitest --run --coverage --config vitest.node.config.mts $(scripts/tests-find.sh --node)",
+    "test:node": "vitest --run --coverage --config vitest.node.config.mts $(scripts/tests-find.sh --node)",
```

Can we keep a default `test` script pointing to `test:node`?

Keeping the default `test` command goes nicely with npm pre/post hooks.

```suggestion
    "test": "pnpm run test:node",
    "test:node": "vitest --run --coverage --config vitest.node.config.mts $(scripts/tests-find.sh --node)",
```

---

Or — we could also just keep it as it was, as "test".

Mind you, npm hooks are integrated with the `pretest` [turbo pipeline](https://github.com/FuelLabs/fuels-ts/blob/981fb625613db6eb3d3d828a2984e0dcc2fd7f3a/turbo.json#L20-L28).

Unless, of course, I'm missing the reason behind this renaming.

---

The name change was done due to the[ ci changes](https://github.com/FuelLabs/fuels-ts/pull/2811/files#diff-245392b692a50c38ecab4381b118862db514035c10983f3bd4f4b7f1f4be4692L42) as a result of the removal of  the `test-ci.sh`. I personally just felt the name `test:node` was more accurate as well since it's only testing in the node environment.  I do take your point about pre/post hooks though, so what do you think about a `test` command that runs both node and browser environments?

```json
 "test": "pnpm run test:node && test:browser"
```

---

Hmm I wouldn't necessarily add browser testing to the default test command.

---

Perhaps `test:all`?
```json
"test:all": "pnpm run --parallel test:node test:browser"
```

---

It could be there, but the issue remains.

My point is about keeping a standard `test` script.

```json
{
    "test": "pnpm run test:node",
    "test:all": "run-p test:node test:browser",
    "test:node": "vitest --run --coverage --config vitest.node.config.mts $(scripts/tests-find.sh --node)",
}
```

---

8939ed94a